[2024-10-25T20:04:09.239+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-25T20:04:09.262+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: inserir_dados_mongodb.insert_data_to_mongo manual__2024-10-25T20:04:00.381488+00:00 [queued]>
[2024-10-25T20:04:09.273+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: inserir_dados_mongodb.insert_data_to_mongo manual__2024-10-25T20:04:00.381488+00:00 [queued]>
[2024-10-25T20:04:09.274+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-25T20:04:09.295+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): insert_data_to_mongo> on 2024-10-25 20:04:00.381488+00:00
[2024-10-25T20:04:09.302+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=764) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-25T20:04:09.305+0000] {standard_task_runner.py:72} INFO - Started process 766 to run task
[2024-10-25T20:04:09.308+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'inserir_dados_mongodb', 'insert_data_to_mongo', 'manual__2024-10-25T20:04:00.381488+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/dag_save_mongodb_v3.py', '--cfg-path', '/tmp/tmpr692ik9x']
[2024-10-25T20:04:09.313+0000] {standard_task_runner.py:105} INFO - Job 51: Subtask insert_data_to_mongo
[2024-10-25T20:04:09.391+0000] {task_command.py:467} INFO - Running <TaskInstance: inserir_dados_mongodb.insert_data_to_mongo manual__2024-10-25T20:04:00.381488+00:00 [running]> on host f31c2b905d22
[2024-10-25T20:04:09.537+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='user' AIRFLOW_CTX_DAG_ID='inserir_dados_mongodb' AIRFLOW_CTX_TASK_ID='insert_data_to_mongo' AIRFLOW_CTX_EXECUTION_DATE='2024-10-25T20:04:00.381488+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-25T20:04:00.381488+00:00'
[2024-10-25T20:04:09.540+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-25T20:04:09.829+0000] {logging_mixin.py:190} INFO - 5 novos registros inseridos com sucesso!
[2024-10-25T20:04:09.870+0000] {logging_mixin.py:190} INFO - Total de documentos na coleção 'clientes': 5
[2024-10-25T20:04:09.871+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-25T20:04:09.914+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-25T20:04:09.915+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=inserir_dados_mongodb, task_id=insert_data_to_mongo, run_id=manual__2024-10-25T20:04:00.381488+00:00, execution_date=20241025T200400, start_date=20241025T200409, end_date=20241025T200409
[2024-10-25T20:04:09.975+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-25T20:04:10.018+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-25T20:04:10.042+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
